### ISSUE TITLE
How to properly set the weights for multi-channel image registration?

### ISSUE THREAD

**Issue by linjing-fang** [embl-cba/elastixWrapper#18]
Hi! I've been recently exploring using Elastix for registering 3-channel 3D stacks and have had a hard time figuring out how to properly set "Weights for multi channel images". The default was given as "1.0, 1.0, 3.0". What does it mean? If i want to register solely based on channel 3, should i set it as (0, 0, 1)?

Thank you for your time!

Best,
Lynn

**Comment by tischi** [comment]
> If i want to register solely based on channel 3, should i set it as (0, 0, 1)?

Yes, exactly: `0,0,1` would only consider channel 3. Does it work as expected?
